diagnostic.md

```diff
@@ -7,40 +7,40 @@ What command do we use to do that? Use the **correct** URL from your fork of
 this repository in your answer.
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+<!-- git clone git@github.com:lisamsmith100/git-diagnostic.git -->
 ```
 
 2.What do you do after cloning a repository, before you start making any
 changes/additions?
 
-<!-- Remove this comment and place your answer here. -->
+<!-- create a branch on which I can do my work -->
 
 3.What command do we use to create a new branch? Name this branch `response`
     in your answer. Then, how do we switch to that branch?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+<!-- git checkout -b response OR git branch response then git checkout response -->
 ```
 
 4.We just wrote some code. What command do we use to see a summary of the
     changes in our working directory?
 
 ```sh
-<!-- Remove this comment and place your answer here. -->
+<!-- git diff -->
```

Comment:
Very close here Lisa! `git diff` will actually show us all of the changes we have made from a file while `git status` will give us a summary of which files have been changed.